src/lua/bcc/bpf.lua

```diff
@@ -55,6 +56,12 @@ function Bpf.static.cleanup()
 
   detach_all("kprobes", Bpf.static.open_kprobes)
   detach_all("uprobes", Bpf.static.open_uprobes)
+
+  for key, perf_buffer in pairs(Bpf.static.perf_buffers) do
+    libbcc.perf_reader_free(perf_buffer)
+    Bpf.static.perf_buffers[key] = nil
+  end
```

Comment:
Not a lua expert. Maybe after all perf_reader_free, just clear Bpf.static.perf_buffers?

Reply:
Yeah, me neither:( I basically mimicked existing code, it compiled and tests pass, but that's all I know......